API/Queries/RootQuery.cs

```diff
@@ -14,7 +15,8 @@ public RootQuery(IServiceProvider serviceProvider)
         {
             var metAPIService = (IMetAPIService) serviceProvider.GetService(typeof(IMetAPIService));
             var metadataService = (IMetadataService) serviceProvider.GetService(typeof(IMetadataService));
-            var populationService = (IPopulationInNorwayService) serviceProvider.GetService(typeof(IPopulationInNorwayService));
+            var populationService =
+                (IPopulationInNorwayService) serviceProvider.GetService(typeof(IPopulationInNorwayService));
 
             Field<ForecastType>("forecast", "Latitude and Longitude defaults to the coordinates of Trondheim",
```

Comment:
Not related to the PR, so this is just a note: this description does not describe the functionality of the query.